### ISSUE TITLE
Applying quantization to the CNN for CIFAR net with BNReLU gives 0.1 validation accuracy

### ISSUE THREAD

**Issue by Abhishek2271** [tensorpack/tensorpack#1543]
### 1. What you did:

(1) Trained the CNN in the /examples/basics for CIFAR net and added quantization layers

### 2. What you observed:

(1) The validation accuracy does not increase above 0.1
![image](https://user-images.githubusercontent.com/33551684/134724509-8b373e7c-87a8-4844-83ad-a2ff4e099642.png)

Hi,

I do not know if this should be asked here but I cannot find references to this anywhere. I trained the /examples/basics/cifar-covnet.py in DoReFaNet and the validation accuracy does not increase above 0.1.

Looking more closely, it seems that the issue is due to the batch normalization layer defined in the convnet.

Is there any specific way the batch normalization layer should be handled during quantization?

![image](https://user-images.githubusercontent.com/33551684/134726508-cd04134d-b5b5-4f45-9138-6b3dd77b000e.png)

**Comment by Abhishek2271** [comment]
it seems that the issue was because I was not using the batch normalization layer on the fc0 layer
